### PR TITLE
Add extra licensing, minutes & support -> docs redirects

### PR DESCRIPTION
**Pull request by hflynn**
When testing the redirect for https://trello.com/c/riJnvMrD/147-downloads-page-cc-license-link I noticed the link behind the "GNU General public license" text was a different construction and only redirecting to the top level About page so this adds that as a redirect to the licensing page too.

### CODE REVIEW

**Comment by hflynn** [comment]
I realize I could also redirect more of the minutes/meetings/ URLs now the older user meetings are live but I'll do that on a follow up PR along with the other items we've identified rather than add even more here.

**Comment by sbesson** [comment]
Deployed on ome-www-dev. The licensing and minutes redirect seems to work but not the support redirects. I will take a look at it tomorrow.

**Comment by hflynn** [comment]
Sorry @sbesson I don't know why this is failing travis - the error message is about nginx and doesn't look related to my changes

File: www/www-deploy.yml [hunk 1]
```diff
@@ -146,8 +146,8 @@
     - match: "~/site/support/?$"
       dest: /docs
 
-    - match: "~/site/support/bio-formats/(?<link>.*[^/])/?$"
-      dest: https://docs.openmicroscopy.org/latest/bio-formats/$link/
+    - match: "~/site/support/bio-formats(?<link>/.*)?$"
+      dest: https://docs.openmicroscopy.org/latest/bio-formats$link;
```
[the remark this answers is not in the document]
Comment (hflynn) [reply www/www-deploy.yml]: Ah, I thought they were a bit random, should have checked with you!

**Comment by hflynn** [comment]
@sbesson passing travis now at least!

**Comment by sbesson** [comment]
Deployed live on www.openmicroscopy.org

```
(ansible) sbesson@ls30630:prod (master) $ pytest playbooks/www/tests/check_redirects.py 
===================================================================================== test session starts =====================================================================================
platform darwin -- Python 2.7.14, pytest-3.4.1, py-1.5.2, pluggy-0.6.0
rootdir: /private/tmp/management_tools/prod, inifile:
plugins: testinfra-1.5.5, pyfakefs-3.3
collected 362 items                                                                                                                                                                           

playbooks/www/tests/check_redirects.py ................................................................................................................................................ [ 39%]
....................................................................................................................................................................................... [ 90%]
...................................                                                                                                                                                     [100%]

================================================================================= 362 passed in 61.73 seconds =================================================================================

```

**Pull request merged**